test/components/__snapshots__/AppItem.spec.jsx.snap

```diff
@@ -22,7 +22,7 @@ exports[`app icon should render correctly 1`] = `
     >
       <a
         data-icon="icon-cozy-drive"
-        href="http://fake.fr"
+        href={"http://fake.fr/"}
```

Comment:
I knew it!

Reply:
;)

Reply:
I did not really understand that, why is there the change ?